packet/packet.go

```diff
@@ -169,6 +177,7 @@ type Packet struct {
 	IPv6     *IPv6Hdr       // Pointer to L3 header in mbuf (must be nil before parsing)
 	TCP      *TCPHdr        // Pointer to L4 header in mbuf (must be nil before parsing)
 	UDP      *UDPHdr        // Pointer to L4 header in mbuf (must be nil before parsing)
+	ICMP     *ICMPHdr       // Pointer to L4 header in mbuf (must be nil before parsing)
 	Data     unsafe.Pointer // Pointer to the packet payload data (invalid before parsing)
 	Unparsed uintptr        // Pointer whole packet data in mbuf (must be non-nil because it is the only public whole element)
```

Comment:
If we have 9 protocols - we at least need to change mbufInit macros inside low.c